packages/langium/src/lsp/default-lsp-module.ts

```diff
@@ -96,7 +96,8 @@ export function createDefaultSharedLSPModule(context: DefaultSharedModuleContext
             FuzzyMatcher: () => new DefaultFuzzyMatcher(),
         },
         workspace: {
-            TextDocuments: () => new NormalizedTextDocuments(TextDocument)
+            TextDocuments: () => new NormalizedTextDocuments(TextDocument),
+            NotebookDocuments: () => new NormalizedNotebookDocuments(TextDocument)
```

Comment:
NotebookDocuments: (services) => new NormalizedNotebookDocuments(services.workspace.TextDocuments)

Reply:
alternatively it seems to work to call
const textDocument = this.sharedServices.workspace.NotebookDocuments.cellTextDocuments.get(uri.toString())
in our service reg.

Reply:
nope. the latter does not work neither

Reply:
cool will try